### DISCUSSION TITLE
在Railway上搭建的waline导入一定量数据后报错

### DISCUSSION THREAD

**Discussion by xiyki** [walinejs/waline#1764] [Q&A] [answered]
由于LeanCloud国际版和vercel在中国大陆地区先后无法正常访问，我在Railway上重新搭建了一个waline<s>*（在自己服务器上搭建几次都没成功）*</s>；我从原先基于LeanCloud和vercel的waline中导出了json文件，在基于Railway上的waline重新导入时发现总是会卡在54/240，并显示错误，不知道是平台本身的限制还是系统Bug；其次，关于独立部署的章节，社区成员是否考虑写的更详细些？
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/83561960/214034630-1f7d268e-70ca-4c8d-97c5-846a1f83506e.png">

**Comment by lizheming** [comment]
1. 可以提供一下导出的数据 check 一下。
2. 关于Leancloud 国际版和 Vercel 的问题看置顶 discussion https://github.com/orgs/walinejs/discussions/1203 https://github.com/orgs/walinejs/discussions/1328
3. 独立部署环节不会补充更详细的文档，因为这本来就不是官方推荐的部署方式，是提供给高端玩家的简单说明，会弄的不需要更多说明，不会弄的有更多说明了还是会不清楚需要官方提供相应的服务咨询。所以官方其实也是用这个文档过滤一部分用户，当然如果你有兴趣欢迎你补充。

**Reply by xiyki** [reply]
> 1. 可以提供一下导出的数据 check 一下。
> 2. 关于Leancloud 国际版和 Vercel 的问题看置顶 discussion https://github.com/orgs/walinejs/discussions/1203 https://github.com/orgs/walinejs/discussions/1328
> 3. 独立部署环节不会补充更详细的文档，因为这本来就不是官方推荐的部署方式，是提供给高端玩家的简单说明，会弄的不需要更多说明，不会弄的有更多说明了还是会不清楚需要官方提供相应的服务咨询。所以官方其实也是用这个文档过滤一部分用户，当然如果你有兴趣欢迎你补充。

1. <s>[文件预览](https://github.com/xiyki/railway-waline/blob/main/waline.json)；*由于涉及到个人隐私，我会在这个问题关闭后将文件删除*</s> 已删除
2. 这个问题我知道，所以才换的Railway😂
3. emm…看来是我不够高端了；不过等我搞明白独立部署之后，确实也会考虑补充

**Comment by xiyki** [comment, marked as answer]
已解决，通过查看Railway中PostgreSQL的Logs发现有一个评论在导入时出现了问题，我将该条评论删除后，再次导入正常
<img width="888" alt="image" src="https://user-images.githubusercontent.com/83561960/214201241-2248afb0-6b72-455e-b828-68809d18f232.png">

该评论如下（ip已处理）：
> 			{
>				"nick": "Anonymous",
>				"ip": "",
>				"mail": "",
>				"ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36",
>				"insertedAt": "2021-03-17T18:32:00.297Z",
>				"isSpam": false,
>				"pid": "600235745582f5630f01333a",
>				"link": "",
>				"comment": "<p><a class=\"at\" href=\"#600235745582f5630f01333a\">@高浩轩 </a> ,  <img alt=\"titter\" referrerPolicy=\"no-referrer\" class=\"vemoji\" src=\"//img.t.sinajs.cn/t4/appstyle/expression/ext/normal/71/2018new_touxiao_org.png\" />何止是图床</p>\n",
>				"url": "/2020/07/08/22/",
>				"QQAvatar": "",
>				"rid": "600235745582f5630f01333a",
>				"isNotified": true,
>				"objectId": "60524b23e061ae461cfcd123",
>				"createdAt": "2021-03-17T18:32:03.649Z",
>				"updatedAt": "2021-03-17T18:32:06.255Z"
>			},
